tensorflow/lite/delegates/gpu/common/transformations/model_transformations.cc

```diff
@@ -66,7 +67,9 @@ bool ApplyGeneralTransformations(ModelTransformer* transformer) {
          transformer->Apply("merge_add_with_convolution",
                             NewMergeAddWithConvolution().get()) &&
          transformer->Apply("merge_mul_with_convolution",
-                            NewMergeMulWithConvolution().get());
```

Comment:
Excess line

Reply:
These are not part of my changes, its already present in the TensorFlow Lite, plus it is not repetition. The above one is for Add and the marked one is for Mul